src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -384,10 +385,14 @@ def transform_summary(
             field='contents.cell_suspensions.total_estimated_cells'
         )
 
+        # Add an organ aggregate based from samples to the ElasticSearch request
```

Comment:
Sometimes it's helpful to summarize the intent before a longer section of code but this comment doesn't convey anything that's not immediately apparent from the code below it.

Reply:
Removed unnecessary comment